numpy/_core/_add_newdocs.py

```diff
@@ -4728,6 +4728,81 @@
     its memory, in which case you can traverse ``a.base`` for a memory handler.
     """)
 
+add_newdoc('numpy._core._multiarray_umath', '_array_converter',
+    """
+    _array_converter(*array_likes)
+
+    Helper to convert one or more objects to arrays.  Integrates machinery
+    to deal with the ``result_type`` and ``__array_wrap__``.
+
+    The reason for this is that e.g. ``result_type`` needs to convert to arrays
+    to find the ``dtype``.  But converting to an array before calling
+    ``result_type`` would incorrectly "forget" whether it was a Python int,
+    float, or complex.
+    """)
+
+add_newdoc('numpy._core._multiarray_umath', '_array_converter', ('scalar_input',
+    """
+    A tuple which indicates for each input whether it was a scalar that
+    was coerced to a 0-D array (and was not already an array or something
+    converted via a protocol like ``__array__()``).
+    """))
+
+add_newdoc('numpy._core._multiarray_umath', '_array_converter', ('as_arrays',
+    """
+    as_arrays(/, subok=True, pyscalars="convert_if_no_array")
+
+    Return the inputs as arrays or scalars.
+
+    Parameters
+    ----------
+    subok : True or False, optional
+        Whether array subclasses are preserved.
+    pyscalars : {"convert", "preserve", "convert_if_no_array"}, optional
```

Comment:
This argument is not used anywhere, so a bit of the same question as for `scalar_input` above. In particular, I wonder if it wouldn't be handier to pass this on *input* (perhaps still allowing to override it here). But without a use case, probably not worth overthinking it now!